code/src/components/TodoList/TodoListItem.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { CategoryIcon } from "../CategoryIcon";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/pro-light-svg-icons";
+import { useDispatch } from "react-redux";
+import { todo } from "reducers/todo";
+import moment from "moment";
+import { grayColor, mainColor } from "../style/colors";
+
+export const TodoListItem = ({ item }) => {
+  const dispatch = useDispatch();
+
+  // Function that sets so that the text changes color depending if their is still time to accomplish the todo or not.
+  const dueDateColor = () => {
+    const dueDateWithoutTime = new Date(item.dueDate);
+    dueDateWithoutTime.setHours(0, 0, 0, 0);
+    const currentDateWithoutTime = new Date();
+    currentDateWithoutTime.setHours(0, 0, 0, 0);
+
+    if (dueDateWithoutTime.getTime() === currentDateWithoutTime.getTime()) {
+      return mainColor;
+    }
+
+    if (item.dueDate <= new Date().getTime()) {
+      return "red";
+    }
+    return grayColor;
+  };
+
+  return (
+    <Item>
+      <CategoryIcon
+        category={item.category}
+        completed={item.completed}
+        canClick={true}
+        id={item.id}
+      />
+      <div>
+        <ItemHeader>{item.title}</ItemHeader>
+        <ItemLocation>{item.place}</ItemLocation>
+        <ItemDueDate style={{ color: dueDateColor() }}>
+          Due date {moment(new Date(item.dueDate)).format(`MM/DD-YYYY`)}
+        </ItemDueDate>
+        <CreatedAt>
+          Created {moment(new Date(item.createdAt)).fromNow()}
+        </CreatedAt>
+      </div>
+      <TrashIcon>
+        {/* When pressing the trashcan button the dispatch call for the removeTodo function in the global state. The payload is the id of the todo. */}
```

Comment:
I think it is really good when you are adding comments to explain what is happening in the code. specially since you often have quite different from other projects. :)